demo_flake8_bugbear.py

```diff
@@ -11,12 +11,14 @@ def printer_is_available(printer_host: str, printer_port: int) -> bool:
             sock.settimeout(0.5)
             sock.connect((printer_host, printer_port))
             return True
-    except:
```

Comment:
In this file, on line 4, there is a line, which should be error with bugbear.

If you have no error, it probably means that you used different installation of flake8.
Try to activate env again and see output of:
`which flake8`
You can force to use flake8 from virtual environment by:
`(venv) $ python -m flake8 demo_flake8_bugbear.py`